devel/swiftlint/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcodeversion 1.0
+
+revision 0
```

Comment:
Can you move this underneath `github.tarball`?

```
github.setup        realm swiftlint 0.52.4
github.tarball_from archive
revision            0
```

Reply:
Moved it!